priv/perf/apps/load_test/lib/application.ex

```diff
@@ -4,24 +4,32 @@ defmodule LoadTest.Application do
   """
   use Application
 
+  alias LoadTest.Connection.ConnectionDefaults
+  alias LoadTest.Ethereum.NonceTracker
+  alias LoadTest.Service.Datadog
+  alias LoadTest.Service.Faucet
+
   def start(_type, _args) do
-    pool_size = Application.fetch_env!(:load_test, :pool_size)
-    max_connections = Application.fetch_env!(:load_test, :max_connection)
+    :ok = start_hackney_pool()
+
+    NonceTracker.init()
```

Comment:
because of the nonce tracking, you can run only one instance of a particula test with the same private key/ethereum account, right? 
let's make sure we're all aware of that.